draft-ietf-lake-edhoc.md

```diff
@@ -579,25 +579,38 @@ In order to reduce round trips and the number of messages, or to simplify proces
 
 EDHOC allows processing of external authorization data (EAD) to be defined in a separate specification, and sent in dedicated fields of the four EDHOC messages (EAD_1, EAD_2, EAD_3, EAD_4). EAD is opaque data to EDHOC.
 
-Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ead_value) as defined below:
+Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ? ead_value) as defined below:
 
 ~~~~~~~~~~~ CDDL
-ead = 1* (
+EAD_x = 1* ead
```

Comment:
> EAD_1 = 1* ead
EAD_2 = 1* ead
EAD_3 = 1* ead
EAD_4 = 1* ead

was where we started 20 hours ago ;-)

Happy to leave this to @cabo and @emanjon

Reply:
I changed my mind about leaving this. We need to merge this soon to progress on other issues.

Reply:
> ? ead, ; EAD_1 

This doesn't work since EAD_1 etc. is one or more ead.

I reverted to having the list of EAD fields in the CDDL, see c27846a.

Unless there is something wrong with the current version, I propose we merge and then people are free to make PRs against that.